blob/blob.go

```diff
@@ -168,6 +179,19 @@ func (b *Bucket) NewRangeReader(ctx context.Context, key string, offset, length
 	return &Reader{r: r}, newBlobError(err)
 }
 
+// Write is a shortcut for creating a Writer via NewWriter and writing p.
+func (b *Bucket) Write(ctx context.Context, key string, p []byte, opt *WriterOptions) error {
```

Comment:
Hm, I'm not sure whether you want to include `WriterOptions` here.

Reply:
Why not? It includes, for example, ContentType.

Reply:
The current options could be fine, but perhaps future ones wouldn't be?

(It's probably fine.)

Reply:
Yea, I think it's fine; the Options are just passed through and I don't anticipate any other code being added here.